dev/com.ibm.ws.org.jboss.weld/src/org/jboss/weld/resources/HotspotReflectionCache.java

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
Please add clear references to where the original source was copied from and clearly mark what you have changed.